front/lib/api/assistant/actions/retrieval.ts

```diff
@@ -388,6 +389,17 @@ export class RetrievalConfigurationServerRunner extends BaseActionConfigurationS
 
     const now = Date.now();
 
+    const uniqueDataSourceViewIds = Array.from(
```

Comment:
nit:  could use `uniq`

Reply:
:+1: done mind checking this is what you had in mind?